connect/reports/renderers/pdf.py

```diff
@@ -30,8 +30,8 @@ def local_fetcher(url, root_dir=None, template_dir=None):
 
 @register('pdf')
 class PDFRenderer(Jinja2Renderer):
-    def render(self, data, output_file):
-        rendered_file = super().render(data, output_file)
+    def generate_report(self, data, output_file):
+        rendered_file = super().generate_report(data, output_file)
         temp_file = f'{output_file}.temp'
         shutil.move(
```

Comment:
Since the render function of the base class use already a temporary directory to generate intermediate artifacts, there is no need to rename the file generated by the superclass.

Reply:
Totally agree